MediaBrowser.Model/Entities/ProviderIdsExtensions.cs

```diff
@@ -49,7 +49,7 @@ public static bool HasProviderId(this IHasProviderIds instance, MetadataProvider
             }
 
             instance.ProviderIds.TryGetValue(name, out string? id);
-            return id;
+            return string.IsNullOrEmpty(id) ? null : id;
```

Comment:
I tend to go with `IsNullOrWhitespace`, cause you never really know if someone added spaces somehow, and stuff isn't stripping the input for whitespace.